docs/api/import_materials.md

```diff
@@ -63,3 +64,138 @@ Content-Type application/json
    ]
 }
 ```
+
+## Import a GEO material:
+
+**Request**:
+
+`POST` `/materials/import/`
+
+Parameters:
+
+Name                           | Type   | Description
+-------------------------------|--------|---
+import_type                    | string | The type of resource being imported. Options include: ["SRA", "GEO", "PROTOCOLS.IO"].
+study_accession                | int    | The accession code for the study represeting the SRP material, in the form "SRP123456".
```

Comment:
This is for SRA, however I'm not actually sure we need a separate section for GEO. I think having one section titled `Import a Dataset` would be sufficient because SRA and GEO aren't really all that different from the user's perspective.

Reply:
Gotcha. Just merged the two sections.